elements/castbar.lua

```diff
@@ -117,6 +121,8 @@ local function UpdatePips(element, numStages)
 	local stageMaxValue = element.max * 1000
 	local isHoriz = element:GetOrientation() == 'HORIZONTAL'
 	local elementSize = isHoriz and element:GetWidth() or element:GetHeight()
+	element.numStages = numStages
+	element.curStage = 0 -- NOTE: Updates only if the PostUpdateStage callback is present
```

Comment:
missing docs for these two

Reply:
You mean like adding them to the attributes list?

Reply:
😖